packages/graphql/src/public/server.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
this map helps the server look-up the specific query contents for the hash it received from the client